pkg/console/operator/sync_v400.go

```diff
@@ -71,12 +71,31 @@ func (co *consoleOperator) sync_v400(ctx context.Context, controllerContext fact
 		return statusHandler.FlushAndReturn(routeErr)
 	}
 
+	authnConfig, err := co.authnConfigLister.Get("cluster")
+	if err != nil {
+		return statusHandler.FlushAndReturn(err)
+	}
+
+	var authServerCAConfig *corev1.ConfigMap
+	switch authnConfig.Spec.Type {
+	case configv1.AuthenticationTypeOIDC, configv1.AuthenticationTypeNone:
+		if len(authnConfig.Spec.OIDCProviders) > 0 {
+			oidcProvider := authnConfig.Spec.OIDCProviders[0]
+			authServerCAConfig, err = co.configMapLister.ConfigMaps(api.OpenShiftConfigNamespace).Get(oidcProvider.Issuer.CertificateAuthority.Name)
+			if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
if NotFound does that imply that we its still in the sync process ?

Reply:
this checks the configuration (OpenShiftConfigNamespace), not the synced image